docsk8s/setting-up/ls-k8s-secure.asciidoc

```diff
@@ -5,19 +5,234 @@ WARNING: This documentation is still in development and may be changed or remove
 
 In order to prepare your environment to be production ready, you'll need to set up secure communication between each of your Elastic resources.
 
-[[security-set-up-tls]]
+[[security-tls]]
 ==== Setting up TLS
 
-It's important to set up transport layer security (TLS) to ensure safe communication between the Elastic Stack components running in Kubernetes.
+It's important to set up transport layer security (TLS) to ensure safe communication between the Elastic Stack components running in {k8s}.
+Let's take {filebeat} and {ls} TLS mutual verification as an example. {ls} serves as the server side, while {filebeat} is the client.
```

Comment:
I wonder if we should either link to the [securing communication between beats and logstash page](https://www.elastic.co/guide/en/beats/filebeat/current/configuring-ssl-logstash.html) or have the same caveat that "Creating a correct SSL/TLS infrastructure is outside the scope of this document" that that document does

Reply:
👍 link to filebeat doc